README.md

```diff
@@ -131,16 +131,16 @@ around:
 ```js
 // main.js
 const worker = new Worker('worker.js');
-const doStuff = Comlink.proxy(worker);
-await doStuff(result => console.log(result));
+const doStuffInWorker = Comlink.proxy(worker);
+await doStuffInWorker(result => console.log(result));
 ```
 
 ```js
 // worker.js
-Comlink.expose(async function (f) {
-  const result = /* omg so expensive */;
-  /* f is a proxy, as if created by proxy(). So we need to use `await. */
-  await f(result);
+Comlink.expose(async function (callback) {
+  const result = await doSomethingTimeConsuming();
```

Comment:
I’d prefer no `await` here as it is perfectly okay (and even one of the motivations) to use blocking code in workers